tests/x11/mate_reconfigure_openqa.pm

```diff
@@ -18,6 +18,9 @@ use utils;
 sub run {
     my $self = shift;
 
+    # Installation take some time and screenlock may appear
+    ensure_unlocked_desktop;
+
```

Comment:
The problem is that this module checks for the desktop (even if not stated explicitly in this module), and it is locked. IMO, there is no other place to add it.

Reply:
sorry, not convinced

Reply:
@okurz what do you propose, then?

Reply:
I think the problem is actually in https://openqa.opensuse.org/tests/831679#step/consoletest_finish/11 where I assume we call `ensure_unlocked_desktop` already but the same symptom as for gnome shows up but that however is not considered for icewm. In https://openqa.opensuse.org/tests/831679#step/boot_to_desktop/4 you can see the clock in the bottom right showing 08:25:54, in https://openqa.opensuse.org/tests/831679#step/consoletest_finish/11 we see 08:26:03 which I doubt is correct because certainly the steps in between took longer so what we see is not actually the right screen content but some old screen content which is falsely matched and accepted as "unlocked desktop" so shortly after in https://openqa.opensuse.org/tests/831679#step/mate_reconfigure_openqa/2 the screen is locked which confuses the tests. So what we need to do is either work on https://progress.opensuse.org/issues/40490 or the according related tickets or apply the same problem for icewm as for gnome, to double check that the screen is really unlocked.

Reply:
Ok, I will try to fix ```ensure_unlocked_desktop``` for mate then.

Reply:
@okurz, done.

Reply:
Does this actually work? https://openqa.opensuse.org/tests/831679#step/consoletest_finish/11 shows an icewm session, not mate. Have you tried out that change?

Reply:
@okurz, indeed, DESKTOP var was still set to mate from previous run. PR update accordingly then.